conmon-rs/server/src/terminal.rs

```diff
@@ -297,7 +297,7 @@ mod tests {
         assert!(!sut.path().exists());
 
         // Write to the slave
-        let mut file = unsafe { fs::File::from_raw_fd(res.slave) };
+        let mut file = unsafe { fs::File::from_raw_fd(res.slave.as_raw_fd()) };
```

Comment:
The `File` will take ownership of the file descriptor with `from_raw_fd`. So it should be `res.slave.into_raw_fd()`.

But `OwnedFd` also implements `Into<File>`:
```suggestion
        let mut file: fs::File = res.slave.into();
```

Reply:
Fixed as suggested.